### PR TITLE
public testnet instructions

### PR DESCRIPTION
**Pull request by silky**
Fixes SCP-3021

I think the instructions need a note to wait for the chain-index to say it's ready to go; otherwise the entire thing fails, silently.

TODO:

- [x] Put the private testnet intructions in here as well
- [x] Verify them again
- [ ] ~Add a gif showing it in operation~
- [ ] ~Add Jann's notes about why it may go wrong.~

Pre-submit checklist:
- Branch
    - [ ] Tests are provided (if possible)
    - [x] Commit sequence broadly makes sense
    - [x] Key commits have useful messages
    - [x] Relevant tickets are mentioned in commit messages
    - [ ] Formatting, materialized Nix files, PNG optimization, etc. are updated
- PR
    - [x] Self-reviewed the diff
    - [x] Useful pull request description
    - [x] Reviewer requested

### CODE REVIEW

File: plutus-pab/test-node/README.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+  - Save this in a file named `restore-wallet.json` in the `testnet` folder:
+
+  ```
+  > cd $PLUTUS/plutus-pab/test-node/
+  > cat testnet/restore-wallet.json
+  { "name": "PAB testing wallet"
+  , "mnemonic_sentence": ["word1", "word2", ...]
+  , "passphrase": "pab123456789"
+  }
```
Comment (koslambrou) [inline plutus-pab/test-node/README.md:66]: Maybe a better way to put the recovery phrase? I had to manually add the double quotes and commas..

File: plutus-pab/test-node/README.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+    localhost:8090/v2/wallets
+  ```
+
+  Then, take note of the Wallet ID that is created, and save that:
+
+  ```
+  > export WALLET_ID=...
+  ```
+
+  - Put some tAda in it, by first obtaining an address:
```
Comment (koslambrou) [inline plutus-pab/test-node/README.md:84]: Correction: `Ada`

Comment (koslambrou) [reply plutus-pab/test-node/README.md:84]: Ah, unless `tAda` means `testnet Ada`

File: plutus-pab/test-node/README.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+  You can verify that the transfer has taken place by clicking the link the
+  faucet application gives you; you see a transaction like so:
+
+  ![](faucet-example.png)
+
+5. Start the chain index:
+
+  ```
+  > cd $PLUTUS/plutus-pab/test-node/
+  > cabal exec -- plutus-chain-index --config testnet/chain-index-config.json start-index
```
Comment (koslambrou) [inline plutus-pab/test-node/README.md:106]: I'm getting a `testnet/chain-index-config.json` not found.

Comment (silky) [reply plutus-pab/test-node/README.md:106]: @koslambrou It should be there; did you run that command from the right dir? (note the file is added in this diff)

File: plutus-pab/test-node/README.md (hunk 1 above)
Comment (silky) [reply plutus-pab/test-node/README.md:66]: Yes; I ended up using a search-and-replace: ` ` to `", "` 😀

File: plutus-pab/test-node/README.md (hunk 2 above)
Comment (silky) [reply plutus-pab/test-node/README.md:84]: Yes, I discovered this yesterday!

File: plutus-pab/test-node/README.md (hunk 1 above)
Comment (koslambrou) [reply plutus-pab/test-node/README.md:66]: Oh right, that works too :D

File: plutus-pab/test-node/README.md (hunk 3 above)
Comment (koslambrou) [reply plutus-pab/test-node/README.md:106]: Oops, I was on the wrong branch

File: plutus-pab/test-node/README.md (hunk 1 above)
Comment (silky) [reply plutus-pab/test-node/README.md:66]: I did do it manually the first few times 😁

**Pull request merged**